### PR TITLE
PreChatSurveyPane Toggle Input Styles

### PR DESCRIPTION
**Pull request by xTEddie**
### Description
- Add the ability to reposition the input toggle positioning

## Solution Proposed
As described above

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

**Review by akhiloct1510 — APPROVED** [review]

File: chat-components/src/components/prechatsurveypane/common/defaultProps/defaultStyles/defaultPreChatSurveyPaneToggleInputStyles.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { IPreChatSurveyPaneToggleInputStyles } from "../../../interfaces/IPreChatSurveyPaneToggleInputStyles";
+
+export const defaultPreChatSurveyPaneToggleInputStyles: IPreChatSurveyPaneToggleInputStyles = {
+    alignItems: "center",
```
Comment (charliewang95) [inline chat-components/src/components/prechatsurveypane/common/defaultProps/defaultStyles/defaultPreChatSurveyPaneToggleInputStyles.ts:4]: nit: remove comma

File: chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx [hunk 2]
```diff
@@ -104,7 +104,10 @@ function PreChatSurveyPane(props: IPreChatSurveyPaneProps) {
                 padding-top: 7px;
                 padding-bottom: 7px;
             }
-            .ac-pushButton { 
+            .ac-input.ac-toggleInput {
+                align-items: ${props.styleProps?.customToggleInputStyleProps?.alignItems ?? defaultPreChatSurveyPaneStyles.customToggleInputStyleProps?.alignItems} !important;
```
Comment (charliewang95) [inline chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx:108]: Please add `customToggleInputStyleProps` to the corresponding place in the dev guide doc

Comment (charliewang95) [inline chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx:108]: Reminder to update change log

Comment (elopezanaya) [reply chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx:108]: +1

Comment (elopezanaya) [reply chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx:108]: +1

**Review by elopezanaya — CHANGES_REQUESTED** [review]
ditto with comments made by Charlie, ensure you have added documentation and changelog

File: chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx (hunk 2 above)
Comment (kevinlin98z) [reply chat-components/src/components/prechatsurveypane/PreChatSurveyPane.tsx:108]: Please share the similar below information in the dev guide doc:
Using baseline would put the checkbox on top, using flex-end would put the checkbox at the bottom, and center is the default value.

**Review by elopezanaya — APPROVED** [review]

**Pull request merged**